vividus-plugin-datetime/src/main/java/org/vividus/bdd/expression/EpochExpressionProcessors.java

```diff
@@ -32,7 +32,7 @@ public EpochExpressionProcessors(DateUtils dateUtils)
     {
         super(List.of(
             new UnaryExpressionProcessor("toEpochSecond",
-                arg -> String.valueOf(dateUtils.toEpochSecond(arg, DateTimeFormatter.ISO_DATE_TIME))),
+                arg -> String.valueOf(dateUtils.parseDateTime(arg, DateTimeFormatter.ISO_DATE_TIME).toEpochSecond())),
             new UnaryExpressionProcessor("fromEpochSecond",
                 arg -> DateTimeFormatter.ISO_DATE_TIME.format(dateUtils.fromEpochSecond(parseLong(arg))))
             ));
```

Comment:
don't you mind to add docs for these expressions?

Reply:
that will be added in another PR